src/applications/registry.scaffold.json

```diff
@@ -132,11 +132,6 @@
     "widgetType": "pension-how-do-i-apply-widget",
     "rootUrl": "/pension-how-do-i-apply-widget"
   },
-  {
```

Comment:
Note: This file is no longer used by the system or build. Unknown why this has not been deleted.